mote/templates/aboutpage.html

```diff
@@ -0,0 +1,743 @@
+                        <li class="nav-item" id="loader"></li>
+                        <li class="nav-item">
+                            <a class="nav-link" aria-current="page" data-bs-toggle="modal" data-bs-target="#chanmode"
+                                type="button" onclick="populate_channel_list();">
+                                Meetings by Channel
+                            </a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" type="button" href="#">
+                                Meetings by Group
+                            </a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" type="button" href="/about">
+                                About
+                            </a>
+                        </li>
+                    </ul>
+                    <div class="d-flex flex-column justify-content-center align-items-center">
+                        <div class="input-group">
+                            <input type="text" autocomplete="off" class="form-control"
+                                placeholder="Search meeting logs"
+                                aria-label="Search meeting logs" aria-describedby="findtext"
+                                style="width: 350px;" id="findtext">
+                        </div>
+                        <div class="input-group" data-bs-auto-close="true">
+                            <ul class="dropdown-menu w-100" id="dropdown">
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </nav>
+        <div class="container-lg">
+            <div class="card text-center mt-1">
+                <div class="card-body pt-0">
+                    <div class="text-center">
+                        <div class="mt-4">
+                            <h1 class="abouthead"><u>Mote 2.0</u></h1>
+
+                            <h4>Flask project used to aggregate and distribute IRC meeting minutes and logs for the Fedora Project</h4>
+                        
+                             <h2 class="abouthead"><u>Description</u></h2>
+                
+                            <p>
+                            Mote is a web application purposed to aggregate and distribute the minutes and logs for IRC/Matrix meetings of the Fedora Project community
+                            The project allows looking up meeting info with channel names, dates/times of occurrence and lists recent meetings for quick access
+                            The backend is written in Python with the use of libraries such as Flask micro web framework, BeautifulSoup4, Urllib3, Click, Werkzeug etc
+                            The frontend is written in HTML5, CSS3 and ES6 with the use of libraries such as Bootstrap 5, jQuery, Popper.JS, EasyQRCode etc
```

Comment:
Add punctuation at the end of sentences. This paragraph is a bit hard to read without it.
Maybe some proper line breaks would help too.

Reply:
sure